flink-ml-core/src/main/java/org/apache/flink/ml/common/window/Windows.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.common.window;
+
+/**
+ * A {@link Windows} determines how input data stream would be sliced into batches and fed into a
```

Comment:
Note that a Flink ML stage takes lists of DataStream as input, which will not be affected by `windows`. So it might be confusing to say that windows affect the input of a Flink stage.

How about the following doc:

Windowing strategy that determines how to create mini-batches from input data.